scout/server/blueprints/variant/controllers.py

```diff
@@ -21,10 +21,11 @@
 )
 from scout.server.blueprints.variant.utils import update_representative_gene
 from scout.server.extensions import cloud_tracks, gens
-from scout.server.links import ensembl, get_variant_links
```

Comment:
ensembl was unused, could have been removed in a separate commit, sorry for the confusion